components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -9,10 +9,8 @@ import {trackEvent} from 'actions/telemetry_actions';
 import BlockableLink from 'components/admin_console/blockable_link';
 import AlertBanner from 'components/alert_banner';
 
-import privateCloudImage from 'images/private-cloud-image.svg';
-import freeTrialPrivateCloudImage from 'images/free-trial-private-cloud-image.svg';
```

Comment:
Nit: Can we remove these svg files? It doesn't look like they're used elsewhere in webapp.